src/js/components/Box.js

```diff
@@ -62,6 +62,7 @@ export default class Box extends Component {
     this._addPropertyClass(classes, CLASS_ROOT, 'separator');
     this._addPropertyClass(classes, CLASS_ROOT, 'size');
     this._addPropertyClass(classes, CLASS_ROOT, 'textAlign', 'text-align');
+    this._addPropertyClass(classes, CLASS_ROOT, 'width');
```

Comment:
@ericsoderberghp good point. Should we combine them width into size or vice versa?